src/client/pages/Project/PullRequest/Layouts/LayoutDeveloper.js

```diff
@@ -7,10 +7,9 @@ import CategoryModule from './common'
 
 const CATEGORIES = [
   { url: 'summary', name: 'Summary' },
-  { url: 'discussion', name: 'Discussion' },
+  { url: 'diff', name: 'Files' },
```

Comment:
I intentionally renamed to Files, don't like `diff` as it sort of incomplete, maybe `Files diff`, or just `Files` :)